src/Compositor.cpp

```diff
@@ -29,7 +29,8 @@ void handleUnrecoverableSignal(int sig) {
         return;
     }
 
-    CrashReporter::createAndSaveCrash(sig);
+    if (g_pCompositor->m_bShouldSaveCrash)
```

Comment:
honestly this is not a great idea. It'd be better to remove the SEGV/ABRT signal handlers altogether in cleanup() probably

Reply:
Maybe this previous commit is closer to that? https://github.com/hyprwm/Hyprland/pull/4811/commits/530d71c7857e2998a80bd1f06fe4a95adeb17a88
I thought that wasnt good if we still want to receive the signal.

Reply:
do it inside cleanup and that'd be my idea

Reply:
thats added. though the core dump still exists probably